src/renderers/dom/client/syntheticEvents/SyntheticEvent.js

```diff
@@ -160,6 +160,7 @@ assign(SyntheticEvent.prototype, {
     this.dispatchConfig = null;
     this._targetInst = null;
     this.nativeEvent = null;
+    this.target = null;
```

Comment:
Ah yea, good call. Really `target` should be in that interface too but I get why it's not.